src/App.js

```diff
@@ -1,5 +1,43 @@
+import { Console } from '@woowacourse/mission-utils';
+import { PROMPT } from './constants/constants';
+import { get3DigitRandom } from './utils/get3DigitRandom';
+import { getStrikeBall } from './utils/getSrikeBall';
+import { validate3Digit, validateIsPlay } from './utils/validateInput';
+
 class App {
-  async play() {}
+  #isPlay = true;
+  #isInput;
+
+  async play() {
+    while (this.#isPlay) {
+      this.#isInput = true;
+      Console.print(PROMPT.startGame);
+      const computer = get3DigitRandom();
+      while (this.#isInput) {
+        const user = String(await Console.readLineAsync(PROMPT.inputNumber));
+        validate3Digit(user);
+        const baseball = getStrikeBall(computer, user);
+        await this.printResult(baseball.ball, baseball.strike);
+      }
+    }
+  }
+
+  async printResult(ball, strike) {
+    if (strike >= 3) {
+      this.#isInput = false;
+      Console.print(`3${PROMPT.strike}`);
+      Console.print(PROMPT.endGame);
+      const user = await Console.readLineAsync(PROMPT.restartOrExit);
+      this.#isPlay = validateIsPlay(user);
+    } else if (ball === 0 && strike === 0) {
+      Console.print(PROMPT.nothing);
+    } else {
+      const result = [];
+      if (ball > 0) result.push(`${ball}${PROMPT.ball}`);
+      if (strike > 0) result.push(`${strike}${PROMPT.strike}`);
```

Comment:
```suggestion
      const result = [ball > 0 ? `${ball}${PROMPT.ball}` : '', strike > 0 ? `${strike}${PROMPT.strike}` : ''];
```
이런식으로도 코드를 줄일 수 있을 것 같습니다!

Reply:
생각지 못했는데 조언 감사합니다!
덕분에 이렇게 더 줄일 수 있었습니다!

```suggestion
      const result = [
        ball > 0 && `${ball}${PROMPT.ball}`,
        strike > 0 && `${strike}${PROMPT.strike}`,
      ];
```